dev/docker-compose.yaml

```diff
@@ -32,13 +32,17 @@ services:
       - DATABASE=be
     profiles:
       - be
+    depends_on:
```

Comment:
why was this moved from the `mongo_seed` common template?

Reply:
It gives an error on docker-compose v2.24:
> service "mongodb_seed" can't be used with `extends` as it declare `depends_on`